app/controllers/keywords_controller.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+class KeywordsController < ApplicationController
+  def create
+    form = CsvUploadForm.new
+
+    if form.save create_params[:file]
```

Comment:
Again raising the concern of not using parenthesis consistently. Since you use them with I18n.t, this should be:

```suggestion
    if form.save(create_params[:file])
```

I recommend you to stick to using parenthesis to avoid any inconsistency issues 🤞

Reply:
Remove parentheses of I18n eb74e685b4c239c4ccfafb68461b3be159e4a296

Reply:
I thinks it's too late to go back to the no parens style now 😥 . Just to be clear, as long as we stick to one style and be consistent, we should be ok, right?

Reply:
@Goose97 in Ruby we prefer the parenthesis to reduce the confusion, the lining tool could help you to add all nessesarry parenthesis automatically, you can do it by

1. Update the `rubocop.yml`
```
Style/MethodCallWithArgsParentheses:
  Enabled: true
```
2. Run `rubocop -a`

More information you can check on https://www.rubydoc.info/gems/rubocop/RuboCop/Cop/Style/MethodCallWithArgsParentheses

Reply:
I also created an RFC for that https://github.com/nimblehq/compass/discussions/1315 so if the team vote for Yes, we can enforce it by modify the rubocop.yml in the Rails template ;)

Reply:
yeah, I prefer the parentheses as well. It's also the Elixir's choice regarding this matter.

I thought that Ruby prefers the parentheses-less style (don't know where I got this idea 😅) and stuck with it from the beginning

Reply:
@Goose97 let's force the parenthesis by update the rubocop.yml file and run the `rubocop -a` on this PR 👌

Reply:
One small thing. Does this rule apply to spec files as well?

Reply:
Fixed in 8aca492dbd8f4380a2e7f1c58c429fad1088359c

Reply:
> Does this rule apply to spec files as well?

@Goose97 let apply it there as well 👍

Reply:
all done 💪